.circleci/config.yml

```diff
@@ -0,0 +1,204 @@
+          background: true
+          when: always
+      - run:
+          name: Wait for Emulator
+          command: source .circleci/avd_helper.sh && waitForAVD
+          when: always
+
+      - run:
+          name: Test Hybrid Local App
+          command: cd .circleci && fastlane android type:hybrid_local
+          when: always
+          no_output_timeout: 900
+      - store_artifacts:
+          path: tmp*/hybrid_localandroidApp/platforms/android/app/build/outputs/apk/
+          destination: Hybrid_Local
+      - store_artifacts:
+          path: Android/app/build/reports/androidTests/connected/
+          destination: Hybrid_Local
+      - store_artifacts:
+            path: /home/circleci/.fastlane/latest_crash.json
+            destination: Hybrid_Local
+      - store_artifacts:
+            path: Android/screenshots/hybrid_local/login.png
+            destination: Hybrid_Local
+      - store_test_results:
+          path: Android/app/build/outputs/androidTest-results/
+
+      - run:
+          name: Test Hybrid Remote App
+          command: cd .circleci && fastlane android type:hybrid_remote
+          when: always
+          no_output_timeout: 900
+      - store_artifacts:
+          path: tmp*/hybrid_remoteandroidApp/platforms/android/app/build/outputs/apk/
+          destination: Hybrid_Remote
+      - store_artifacts:
+          path: Android/app/build/reports/androidTests/connected/
+          destination: Hybrid_Remote
+      - store_artifacts:
+          path: /home/circleci/.fastlane/latest_crash.json
+          destination: Hybrid_Remote
+      - store_artifacts:
+          path: Android/screenshots/hybrid_remote/login.png
+          destination: Hybrid_Remote
+      - store_test_results:
+          path: Android/app/build/outputs/androidTest-results/
+
+
+#  test-ios:
+#      <<: *mac
+#      steps:
+#        - checkout
+#        - run: *setup
+#        - run: *install-ant
+#        - run:
+#            name: Build Native Objective-C App
+#            command: ./test/test_force.js --os=ios --apptype=native
+#            when: always
+#        - run:
+#            name: Build Native Swift App
+#            command: ./test/test_force.js --os=ios --apptype=native_swift
+#            when: always
+#        - run:
+#            name: Build Hybrid Local App
+#            command: ./test/test_force.js --os=ios --apptype=hybrid_local
+#            when: always
+#        - run:
+#            name: Build Hybrid Remote App
+#            command: ./test/test_force.js --os=ios --apptype=hybrid_remote
+#            when: always
+#        - run:
+#            name: Build React Native App
+#            command: ./test/test_force.js --os=ios --apptype=react_native
+#            no_output_timeout: 20m
+#            when: always
+
+workflows:
+  version: 2
+
+  pr-build-all-apps:
+    jobs:
+      - test-android
+
+  # Cron are on a timezone 8 hours ahead of PST
+  # Build everything on Saturday Afternoon
+  weekly-build-all-apps:
```

Comment:
Run weekly.  We can update this later to run different things every week (debug vs release or sfdx cli).